### PR TITLE
[SDPA-4714] card promo config

### PR DESCRIPTION
**Pull request by rubyllamadora**
**JIRA issue:** https://digital-engagement.atlassian.net/browse/SDPA-4714

### Changed

1.  Added Tide config to map card promo component
2. Moved common css styling for both nav and promo components into cards.scss to easily manage

### Screenshots

image

### CODE REVIEW

File: packages/components/Molecules/Card/CardPromo.vue [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <template>
-  <rpl-link class="rpl-card-promo" :class="modifiers" :href="link.url">
```
Comment (tim-yao) [inline packages/components/Molecules/Card/CardPromo.vue:2]: We should keep `rpl-card-promo` class with modifiers if we follow `BEM`. See examples: http://getbem.com/naming/

File: packages/components/Molecules/Card/CardPromo.vue [hunk 2]
```diff
@@ -38,15 +38,20 @@ export default {
...
+    image: Object,
+    dateStart: String,
+    dateEnd: String,
     tag: String,
     topic: String,
     status: String,
     title: String,
     summary: String,
-    link: Object
+    link: Object,
```
Comment (tim-yao) [inline packages/components/Molecules/Card/CardPromo.vue:49]: Does link a mandatory field for card? If it does, should we have a validation?

File: packages/components/Molecules/Card/CardPromo.vue (hunk 1 above)
Comment (rubyllamadora) [reply packages/components/Molecules/Card/CardPromo.vue:2]: I've moved this to the modifiers() as it confuses me having two `class=""`. I'll put it back here so it's more clear that it's only for modifiers 👍

File: packages/components/Molecules/Card/CardPromo.vue (hunk 2 above)
Comment (rubyllamadora) [reply packages/components/Molecules/Card/CardPromo.vue:49]: I probably need to review this props declaration e.g. provide default values if it's not mandatory and explicitly declare required fields.

**Review by tim-yao — APPROVED** [review]

**Comment by rubyllamadora** [comment]
This has been QA'ed and sub-tickets created by QA for both Promo and Nav bugs. Merging to parent ticket.

**Pull request merged**